sessionctx/variable/varsutil_test.go

```diff
@@ -665,4 +675,6 @@ func (s *testVarsutilSuite) TestConcurrencyVariables(c *C) {
 	c.Assert(vars.indexLookupConcurrency, Equals, ConcurrencyUnset)
 	c.Assert(vars.IndexLookupConcurrency(), Equals, exeConcurrency)
 	c.Assert(vars.WindowConcurrency(), Equals, wdConcurrency)
+	c.Assert(vars.StreamAggConcurrency(), Equals, saConcurrency)
+
```

Comment:
Please remove this unnecessary blank line.

Reply:
679 is the last line in the file, it should be added automatically....